erpnext/patches/v13_0/set_coa_batch_in_batch.py

```diff
@@ -0,0 +1,12 @@
+import frappe
+
+def execute():
+    """
+    Set coa batch value in batch document if batch and coa batch is set in package tag
+    """
+    frappe.reload_doc("stock", "doctype", "batch")
+    package_tags = frappe.get_all("Package Tag",
+        filters = [{'batch_no': ["is", "set"], "coa_batch_no": ["is", "set"]}],
+        fields = ['name', 'batch_no', "coa_batch_no"])
```

Comment:
- we are not using name anywhere right, we can skip that
```suggestion
        fields = ['batch_no', "coa_batch_no"])
```